README.rst

```diff
@@ -32,11 +32,9 @@ See `doc/install.txt`_ in the source code archive for general information. Excep
 
 .. _doc/install.txt: doc/install.txt
 
-Python 3 or later is needed.
+Python 3.5.0 or later is needed.
 
-The version in the pip repository is old. Instead, you can use pip to install the latest release from git: ``pip install git+https://github.com/linkchecker/linkchecker.git@v9.4.0``. See `#4 <https://github.com/linkchecker/linkchecker/pull/4>`_.
-
-Windows builds are seriously lagging behind the Linux releases, see `#53 <https://github.com/linkchecker/linkchecker/issues/53>`_ for details. For now, the only two options are to install from source or use `Docker for Windows <https://www.docker.com/docker-windows>`_.
+You can use pip to install the latest from git: ``pip3 install git+https://github.com/linkchecker/linkchecker.git``. See `#4 <https://github.com/linkchecker/linkchecker/pull/4>`_.
```

Comment:
Drop the `See #4`, it's about releasing 9.4.0, which is already out there.  We were offering git installs to overcome the problem in 9.3 where it would fail on startup with any recent requests version due to the busted version check.  Now we're offering git installs to overcome the problem in 9.4 where it will not work Python 3.

Reply:
Done.